### PR TITLE
Add support for Mastodon to developer profiles

### PR DESCRIPTION
**Pull request by jordelver**
Add support for Mastodon to developer profiles

2023-02-10-16 49 39@2x

2023-02-10-16 47 48@2x

Icon was sourced from https://joinmastodon.org/branding and run through https://jakearchibald.github.io/svgomg/ before being manually edited to include the title.

## Pull request checklist

- [x] My code contains tests covering the code I modified
- [x] I linted and tested the project with `bin/check`
- [x] I added significant changes and product updates to the [changelog](CHANGELOG.md) -- Not sure if this qualifies as "significant" 😄

### CODE REVIEW

**Comment by joemasilotti** [comment]
This looks great! A few requests before I merge it in:

1. ~Merge in `main`.~ Done: I did this when playing with the code so why not push it.
2. Add `mastodon` to the "Core developer" seed on `db/seeds/developers.rb:72` with a URL like `website`.
3. Ensure that if someone types in `https://ruby.social/joe` that it will link to that, and not `https://https://ruby.social/joe`. Check out `ExternalLinkComponent` and `LinksHelper` for tips.
4. Finally, please add this to the CHANGELOG. This is definitely a significant change!

**Comment by jordelver** [comment]
Thanks @joemasilotti!

> 2. Add `mastodon` to the "Core developer" seed on `db/seeds/developers.rb:72` with a URL like `website`.

✅ 

> 3. Ensure that if someone types in `https://ruby.social/joe` that it will link to that, and not `https://https://ruby.social/joe`. Check out `ExternalLinkComponent` and `LinksHelper` for tips.

Good idea. I've made this change.

> 4. Finally, please add this to the CHANGELOG. This is definitely a significant change!

Done 👍

**Review by joemasilotti — APPROVED** [review]

**Comment by joemasilotti** [comment]
Excellent! Looks great, thanks for your contribution. Merging in now. :)

**Pull request merged**
